TELCODOCS-1388: Add IPI troubleshooting information to our documentation (#73672)
openshift/openshift-docs: modules/ipi-install-troubleshooting-troubleshooting-the-cluster-network-operator.adoc

// This module is included in the following assemblies: 
//
// installing/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc

:_mod-docs-content-type: PROCEDURE
[id="troubleshooting-the-cluster-network-operator_{context}"]
= Troubleshooting the Cluster Network Operator

> The Cluster Network Operator is responsible for deploying the networking components. It runs very early in the installation process, after the control plane nodes have come up but before the installer removes bootstrap control plane. Issues with this Operator can indicate installer issues.

Author: Done.

.Procedure

> . Ensure the network configuration exists by executing the following command:

Reviewer: ```suggestion
. Ensure the network configuration exists by running the following command:
```

+
[source,terminal]
----
$ oc get network -o yaml cluster
----
+

> If it doesn't exist, the installer didn't create it. To determine why, execute the following command:

Author: Done.

> If it doesn't exist, the installation program didn't create it. To determine why, execute the following command:

Reviewer: ```suggestion
If it doesn't exist, the installation program didn't create it. To find out why, run the following command:
```

+
[source,terminal]
----
$ openshift-install create manifests
----
+

> Review the manifests to determine why the installer didn't create the network configuration.

Author: Done.

> . Ensure the network is running by executing the following command:

Reviewer: ```suggestion
. Ensure the network is running by entering the following command:
```
